recipes/meshplex/meta.yaml

```diff
@@ -0,0 +1,47 @@
+{% set name = "meshplex" %}
+{% set version = "0.13.4" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/nschloe/meshplex/archive/v{{ version }}.tar.gz
+  sha256: ceadd26da8c4bfdd4e543f48944b43a5c38ad4faa55563330651fb6d6f6ac765
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+
+requirements:
+  host:
+    - importlib_metadata
+    - pip
+    - python
+    - numpy >=1.9
+    - scipy
+    - meshio >=4,<5
```

Comment:
You probably don't need anything but pip and python on host. Let's try that before merging this.

Reply:
OK